src/static/swagger.json

```diff
@@ -713,25 +828,57 @@
                 }
             }
         },
-        "/account": {
+        "/presentation/verify": {
             "post": {
                 "tags": [
-                    "Account"
+                    "Presentation"
                 ],
-                "summary": "Create a client",
-                "description": "<p>This endpoint verifies the JWT token and creates a customer if they don't exist</p>",
+                "summary": "Verify a credential presentation",
+                "description": "<p>This endpoint verifies the credential presentation. As input it takes the entire presentation itself</p>",
+                "operationId": "presentation",
                 "security": [
                     {
                         "bearerAuth": []
                     }
                 ],
+                "requestBody": {
+                    "content": {
+                        "application/x-www-form-urlencoded": {
+                            "schema": {
+                                "$ref": "#/components/schemas/PresentationRequest"
+                            }
+                        },
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/PresentationRequest"
+                            }
+                        }
+                    }
+                },
                 "responses": {
                     "200": {
                         "description": "The request was successful",
                         "content": {
                             "application/json": {
                                 "schema": {
-                                    "$ref": "#/components/schemas/Customer"
+                                    "$ref": "#/components/schemas/IVerifyResult"
+                                },
+                                "example": {
+                                    "verified": true,
+                                    "policies": {
+                                        "aud": false,
+                                        "audience": false
+                                    },
+                                    "issuer": "did:key:z6MkqJNR1DHxX2qxqDYx9tNDsXoNRVpaVvJkLPeCYqaARz1n",
+                                    "signer": {
+                                        "controller": "did:key:z6MkqJNR1DHxX2qxqDYx9tNDsXoNRVpaVvJkLPeCYqaARz1n",
+                                        "id": "did:key:z6MkqJNR1DHxX2qxqDYx9tNDsXoNRVpaVvJkLPeCYqaARz1n#z6MkqJNR1DHxX2qxqDYx9tNDsXoNRVpaVvJkLPeCYqaARz1n",
+                                        "publicKeyBase58": "Br7NQy3XBVMViiiFUKQP2SFNbvYj634PeNjGiZc9WmEQ",
+                                        "type": "Ed25519VerificationKey2018",
+                                        "verifier": [
+                                            "did:key:z6MkhAxBJD1rqoVpbupm3CFNvkhvQd7LXtLoipfnWehvXBhE"
```

Comment:
This seems weird/wrong. Why is the issuer a `did:key`? Shouldn't this be `did:cheqd`? Verifier as `did:key` also seems weird. Need to understand whether this is because these are incorrect examples, or actual response.

Reply:
There was an error in swagger, which overrode the example fixed it. Thanks